### DISCUSSION TITLE
How to generate get and setter now? NVIM ot CMD line

### DISCUSSION THREAD

**Discussion by brunojoao** [phpactor/phpactor#2710] [Q&A]
I am using the newest version of phpactor and does not have **code:generate:getter-setter** anymore.

in Astronvim, using **:LspInstall** phpactor on Class doest not have option to get and setter, notthing found. No code action found

![image](https://github.com/user-attachments/assets/f5de7c1e-93a3-417a-88af-1eaec9b54f7e)

![image](https://github.com/user-attachments/assets/c62703ee-7f53-47eb-b928-2bf44c16e3fa)

**Comment by brunojoao** [comment]
![image](https://github.com/user-attachments/assets/985a10f3-5d8b-4e2c-bfc8-1695eb2ebe1d)
Why doesnot work phpactor extension anymore, not listed on phpactor -v

-------------------
Available commands:
![image](https://github.com/user-attachments/assets/965d4ef6-bda3-4e65-bd98-96fd2979ffc6)

**Reply by dantleech** [reply]
Extensions are now all built-in and have been for some years.  I was unaware that we had an action for getters AND setters tbh, I just know "generate accessor"

**Comment by brunojoao** [comment]
Resolved, iam using this file config, and select all variables and code action appears
[phpactor.json](https://github.com/user-attachments/files/16445457/phpactor.json)

**Comment by dantleech** [comment]
yep, and we should probably remove that documentation about extensions :sweat_smile:

**Reply by dantleech** [reply]
@brunojoao where did you find that documentation? was it the PDF download from google?

**Reply by brunojoao** [reply]
i dont remember, but i have search in google by phpactor docs
https://phpactor.readthedocs.io/en/master/extensions/installing.html

**Reply by dantleech** [reply]
ah ok, it's still hosting documentation for the 5 year old master branch, I'll fix that :laughing:

**Reply by dantleech** [reply]
ok, I think I've killed that page :+1:
